modules/apps/29-fee/keeper/migrations_test.go

```diff
@@ -118,10 +118,51 @@ func (suite *KeeperTestSuite) TestMigrate1to2() {
 				suite.Require().Equal(initModuleAccBal.Sub(unusedFee...).Sort(), moduleAccBal)
 			},
 		},
+		{
+			"success: more than one packet",
+			func() {
+				packetFee := types.NewPacketFee(fee, refundAcc.String(), []string(nil))
+				packetFees = []types.PacketFee{packetFee}
+
+				feesToModule := sdk.NewCoins()
+				for _, packetFee := range packetFees {
+					feesToModule = feesToModule.Add(keeper.LegacyTotal(packetFee.Fee)...)
+				}
+
+				// add second packet to have escrowed fees
+				packetID2 = channeltypes.NewPacketID(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 2)
+
+				if !feesToModule.IsZero() {
+					// escrow the packet fee for the second packet & store the fee in state
+					suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID2, types.NewPacketFees(packetFees))
+					err := suite.chainA.GetSimApp().BankKeeper.SendCoinsFromAccountToModule(suite.chainA.GetContext(), refundAcc, types.ModuleName, feesToModule)
+					suite.Require().NoError(err)
+				}
+			},
```

Comment:
```suggestion
				// add second packet to have escrowed fees
				packetID2 = channeltypes.NewPacketID(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 2)

				// escrow the packet fee for the second packet & store the fee in state
				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID2, types.NewPacketFees(packetFees))
				err := suite.chainA.GetSimApp().BankKeeper.SendCoinsFromAccountToModule(suite.chainA.GetContext(), refundAcc, types.ModuleName, keeper.LegacyTotal(packetFee.Fee))
				suite.Require().NoError(err)
			},
```

I would just keep it simple, set two distinct packet fees into state